examples/tutorials/nb_python/ECCV_2020_Interactivity.py

```diff
@@ -846,22 +840,20 @@ def build_widget_ui(obj_attr_mgr, prim_attr_mgr):
 sphere_template_cpy.mass = ball_mass
 obj_attr_mgr.register_template(sphere_template_cpy, "ball")
 
-ball_id = sim.add_object_by_handle("ball")
-set_object_state_from_agent(sim, ball_id, offset=np.array([0, 1.4, 0]))
+ball_obj = rigid_obj_mgr.add_object_by_template_handle("ball")
+set_object_state_from_agent(sim, ball_obj, offset=np.array([0, 1.4, 0]))
 
 # @markdown Initial linear velocity (m/sec):
 lin_vel_x = 0  # @param {type:"slider", min:-10, max:10, step:0.1}
 lin_vel_y = 1  # @param {type:"slider", min:-10, max:10, step:0.1}
 lin_vel_z = 5  # @param {type:"slider", min:0, max:10, step:0.1}
-initial_linear_velocity = mn.Vector3(lin_vel_x, lin_vel_y, lin_vel_z)
-sim.set_linear_velocity(initial_linear_velocity, ball_id)
+ball_obj.linear_velocity = mn.Vector3(lin_vel_x, lin_vel_y, lin_vel_z)
 
 # @markdown Initial angular velocity (rad/sec):
 ang_vel_x = 0  # @param {type:"slider", min:-100, max:100, step:0.1}
```

Comment:
If these are floats, the Vector3d is unnecessary, right?

Reply:
Sometimes they get rounded to an int by the interface (when the sliders are moved), which is why they were built the way they were previously.